src/System.Text.Json/tests/Utf8JsonReaderTests.TryGet.cs

```diff
@@ -522,6 +522,22 @@ public static void InvalidConversion()
                     }
                     catch (InvalidOperationException)
                     { }
+
+                    try
+                    {
+                        Guid value = json.GetGuid();
+                        Assert.True(false, "Expected GetGuid to throw InvalidOperationException due to mismatched token type.");
+                    }
+                    catch (InvalidOperationException)
+                    { }
```

Comment:
I think @ahsonkhan recently added an assert throwing helper for working with Utf8JsonReader.  We should use that instead of open coding these try/catch blocks with Assert.True(false, …).  And if I'm remembering incorrectly and such a helper wasn't added, we should add one :)